### PR TITLE
[zh] sync traffic-management/mirroring

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
This PR syncs page traffic-management/mirroring.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/mirroring/index.md [hunk 1]
```diff
@@ -133,41 +135,93 @@ test: yes
...
+    - name: httpbin-v1
+      port: 80
+EOF
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+1. 现在所有流量都转到 `httpbin:v1` 服务，向服务发送一部分流量：
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/mirroring/index.md:224]:
```suggestion
2) 现在所有流量都转到 `httpbin:v1` 服务，并向此服务发送请求：
```
Do we need follow the style of `2)` same as en? Line 245 has the same issue:
```
1. 分别查看 `httpbin` Pod的 `v1` 和 `v2` 两个版本的日志。您可以看到 `v1` 版本的访问日志条目，而 `v2` 版本没有日志：
```

File: content/zh/docs/tasks/traffic-management/mirroring/index.md [hunk 2]
```diff
@@ -206,34 +260,79 @@ test: yes
...
+{{< /text >}}
+
+This route rule sends 100% of the traffic to `v1`. The `RequestMirror` filter
+specifies that you want to mirror (i.e., also send) 100% of the same traffic to the
+`httpbin:v2` service. When traffic gets mirrored,
+the requests are sent to the mirrored service with their Host/Authority headers
+appended with `-shadow`. For example, `cluster-1` becomes `cluster-1-shadow`.
+
+Also, it is important to note that these requests are mirrored as "fire and
+forget", which means that the responses are discarded.
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/mirroring/index.md:331]: line 324 -331 need be localized too.

File: content/zh/docs/tasks/traffic-management/mirroring/index.md [hunk 3]
```diff
@@ -206,34 +260,79 @@ test: yes
...
+
+{{< /tab >}}
+
+{{< /tabset >}}
 
 1. 发送流量：
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/mirroring/index.md:337]:
```suggestion
2) 发送流量：
```
May need to follow the en style `2)`, which is a special syntax to compile.

File: content/zh/docs/tasks/traffic-management/mirroring/index.md [hunk 4]
```diff
@@ -258,10 +357,27 @@ test: yes
...
+{{< text bash >}}
+$ kubectl delete httproute httpbin
+$ kubectl delete svc httpbin-v1 httpbin-v2
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
 
 1. 关闭 [Httpbin]({{< github_tree >}}/samples/httpbin) 服务和客户端：
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/mirroring/index.md:382]:
```suggestion
2)  关闭 [Httpbin]({{< github_tree >}}/samples/httpbin) 服务和客户端：
```

File: content/zh/docs/tasks/traffic-management/mirroring/index.md (hunk 1 above)
Comment (windsonsea) [reply content/zh/docs/tasks/traffic-management/mirroring/index.md:224]: GG，这一条没改哇，英文的 `2)` 是一种特殊的语法

**Comment by windsonsea** [comment]
编号 貌似还是不太对，还有大段的英文，需要确认一下额

**Comment by Zhuzhenghao** [comment]
Sorry about my mistake that I forgot to push my commits.

**Review by windsonsea — APPROVED** [review]
/lgtm after minor changes

**Comment by windsonsea** [comment]
@zirain need your help to solve the lint issue, thanks.

**Pull request merged**